libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat

```diff
@@ -35,7 +35,7 @@ I2C_ORDER I2C1
 UART_ORDER OTG1 USART3 USART1 USART2 UART4 UART7 USART6
 
 # buzzer
-PD15 TIM4_CH4 TIM4 GPIO(77) ALARM
+PD15 TIM4_CH4 TIM4 GPIO(77) ALARM NODMA
```

Comment:
afaik alarm do not use dma anyway

Reply:
i wanted to make sure of that. this configuration seems to be safe - i did not have a single incident on it related to dshot stream interruption, twitches or flips. something is wrong in the way how dma works when active on serial ports. for now i think it is safer to have nodma everywhere where it is not essential, on f7 at least.

Reply:
please remove the NODMA, it doesn't do anything and will just confuse other people reading the file

Reply:
thx, i will remove it. it would be nice to see any documentation about what codeword is applicable in what situation. i was not aware ALARM was different from any other PWM output.